### PR TITLE
Updating dependencies sync

### PR DESCRIPTION
**Pull request by shermosa**
To keep in sync with main branch https://github.com/klarna/HiveRunner/pull/151

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,10 +16,13 @@ NOTE: This CHANGELOG tracks changes in the 5.x (Hive 2.x) line. For 6.x (Hive 3.
 - Renamed `InsertTestData` in `com.klarna.hiverunner.examples.junit4` to `InsertTestDataTest`.
 - Renamed `SetHiveConfValues` in `com.klarna.hiverunner.examples.junit4` to `SetHiveConfValuesTest`.
 - Updated `surefire-version-plugin` from `2.21.0` to `2.22.0`.
-- Updated `junit.jupiter.version` from `5.6.0` to `5.7.0`.
+- Updated `junit.jupiter.version` from `5.6.0` to `5.7.1`.
+- Updated `mockito-core` from `2.18.3` to `3.8.0`.
+- Updated `junit` from `4.13.1` to `4.13.2`.
```
[the remark this answers is not in the document]
Comment (massdosage) [reply CHANGELOG.md:21]: I noticed this same thing in `main` earlier today, I'll quickly fix it there in the mean time.

[the remark this answers is not in the document]
Comment (massdosage) [reply CHANGELOG.md:21]: @shermosa please make the above look the same as https://github.com/klarna/HiveRunner/commit/4afe17d088b0051b6ce3895a7999c5ebc835ecb8

File: src/test/java/com/klarna/hiverunner/HiveRunnerExtensionTest.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
 @ExtendWith(HiveRunnerExtension.class)
 public class HiveRunnerExtensionTest {
 
-  @HiveSQL(files = {"HiveRunnerExtensionTest/test_query.sql"})
+  @HiveSQL(files = { "HiveRunnerExtensionTest/test_query.sql" })
```
Comment (massdosage) [inline src/test/java/com/klarna/hiverunner/HiveRunnerExtensionTest.java:32]: I don't think we should have the whitespace on either side of that string, we don't do that elsewhere in this project AFAIK.

**Review by massdosage — CHANGES_REQUESTED** [review]

File: src/test/java/com/klarna/hiverunner/HiveRunnerExtensionTest.java (hunk 2 above)
Comment (shermosa) [reply src/test/java/com/klarna/hiverunner/HiveRunnerExtensionTest.java:32]: We don't, but the main branch has them in this file so I wanted to make both branches exactly the same. I'll take them off here but there's still two whitespaces in the main branch. If you want us to be pedantic I can make a PR to take those two whitespaces off the main branch :)

**Review by JayGreeeen — APPROVED** [review]

**Review by massdosage — APPROVED** [review]

**Comment by massdosage** [comment]
Fixes #145

File: src/test/java/com/klarna/hiverunner/HiveRunnerExtensionTest.java (hunk 2 above)
Comment (massdosage) [reply src/test/java/com/klarna/hiverunner/HiveRunnerExtensionTest.java]: I want us to be pedantic so yes please ;P

**Pull request merged**